src/test/java/com/rakovets/course/java/core/practice/oop/principles/cat_home/SiameseTest.java

```diff
@@ -0,0 +1,51 @@
+package com.rakovets.course.java.core.practice.oop.principles.cat_home;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
+
+import java.util.stream.Stream;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+class SiameseTest {
+
+    @Test
+    void mew() {
+        assertEquals("Siamese mew", new Siamese("Siamese").mew());
+    }
+
+    static Stream<Arguments> mewProviderArguments() {
+        return Stream.of(
+                Arguments.of(35.0, new Person(50.0))
+        );
+    }
+
+    @ParameterizedTest
+    @MethodSource("mewProviderArguments")
+    void mew(double expectedDouble, Person person) {
+        Siamese siamese = new Siamese("Siamese");
+        siamese.mew(person);
+        assertEquals(expectedDouble, person.getHappiness());
```

Comment:
You could use 'Given-When-Then' approach.

Reply:
nah.. I like it the way it is